python/GafferUITest/PathListingWidgetTest.py

```diff
@@ -950,6 +950,12 @@ def testModelFirstQueryDoesntEmitDataChanged( self ) :
 			self.assertEqual( model.rowCount(), 1 )
 			self.assertTrue( model.index( 0, 0 ).isValid() )
 
+			if sortable :
+				# Sort by "Value" column as sorting by defaultNameColumn
+				# doesn't trigger a data update.
+				widget._qtWidget().header().setSortIndicator( 1, QtCore.Qt.AscendingOrder )
+				_GafferUI._pathModelWaitForPendingUpdates( GafferUI._qtAddress( model ) )
```

Comment:
Why do we need another wait here? If we want to honour the intentions of the original test, shouldn't we set the sort indicator at L939 and expect that the existing waits are sufficient?

Reply:
Fixed in 442c5f3.